docs/management/index-patterns.asciidoc

```diff
@@ -158,6 +158,9 @@ date values in {es}, you can use a {kib} field formatter to change the display t
 <<field-formatters-geopoint, geopoints>>,
 and <<field-formatters-numeric, numbers>>.
 
+In case you want to customize the displayed field name provided by {es}, you can 
```

Comment:
```suggestion
To customize the displayed field name provided by {es}, you can 
```

Reply:
I also think, we should change this "To edit the field format and popularity counter" to "To edit the field display"

Reply:
Thx, I've applied those changes, and storing in retina resolution leads to such a big image, changed that